sdk/go/common/workspace/loaders.go

```diff
@@ -190,6 +192,7 @@ func LoadProjectStackBytes(
 	b []byte,
 	path string,
 	marshaller encoding.Marshaler,
+	diags diag.Sink,
```

Comment:
Really annoying, but I'd have preferred the sink as the first argument to match convention elsewhere of it being on the first arguments passed in.

Reply:
Ah I thought about this for a moment but dismissed it out of laziness. You're right though, I'll go make it the first argument.